cookbooks/aws-parallelcluster-slurm/files/default/head_node_slurm/epilog

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Cookbook Name:: aws-parallelcluster
+#
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script removes content with the following format to /etc/hosts file
+# #HOSTS_JOB_<JOBID>
+# 192.168.1.2 queue-0-st-compute-resource-0-1 ip-192-168-1-2
+# 192.168.1.10 queue-0-st-compute-resource-0-2 ip-192-168-1-10
+# #END_JOB_<JOBID>
+#
+# SLURM_JOB_ID is a env var provide by Slurm
+
+LOG_FILE_PATH="/var/log/parallelcluster/slurm_prolog_epilog.log"
+
+_log() {
+    text=$1
+    level="${2:-INFO}" # If the second argument is not provided, "INFO" is the default log level
+    log_time=$(date "+%Y-%m-%d %H:%M:%S")
+    echo "${log_time} - ${level} - ${text}" >> "${LOG_FILE_PATH}"
+}
+
+_log "Removing nodes belonging to job ${SLURM_JOB_ID} from /etc/hosts"
+if ! sed_output=$(sed -i "/#HOSTS_JOB_${SLURM_JOB_ID}/,/#END_JOB_${SLURM_JOB_ID}/d" /etc/hosts 2>&1); then
+  _log "Failed to remove nodes belonging to job ${SLURM_JOB_ID}: ${sed_output}" "ERROR"
```

Comment:
Let's keep the 4 spaces indentation convention everywhere in our shells scripts

Reply:
Done